### DISCUSSION TITLE
Suspended Without Feedback or Notification

### DISCUSSION THREAD

**Discussion by Vitor5source** [community/community#145238] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Question

### Body

Hello,

I recently discovered that my GitHub account has been suspended, but I didn’t receive any feedback, explanation, or even an email notifying me of the suspension. It has been over two weeks since I submitted a support ticket, and I still haven’t received any response or clarification.
GitHub is an essential platform for me; I use it for my school projects, study materials, and as a portfolio to showcase my hard work. I’ve invested significant time and effort in creating and maintaining my repositories, and losing access to them is incredibly stressful and discouraging.
I understand that these situations may take time to address, but I was wondering if waiting two weeks is typical for cases like mine, or if there might be any updates on my ticket.
Thank you for your understanding and support.

Best ,
SincooV

**Comment by awade12** [comment]
I had mine suspended today also witihout warning, or anything.

**Comment by j12367385** [comment]
Redmi
